ush/plotting_scripts/plot_date_by_level.py

```diff
@@ -604,7 +604,7 @@
                 ax.xaxis.set_minor_locator(md.MonthLocator())
             else:
                 ax.xaxis.set_minor_locator(md.DayLocator())
-            if ax.is_last_row() \
+            if ax.get_subplotspec().is_last_row() \
```

Comment:
I think I would prefer to keep this out of this PR since these scripts still need to run on WCOSS2 for plotting and it does not have the newer matplotlib available, unless this is backwards compatible.  Have you tested this @tsga?